zanata-adapter-xliff/src/main/java/org/zanata/adapter/xliff/XliffWriter.java

```diff
@@ -216,30 +221,12 @@ public static void writeFile(File file, Resource doc, String locale,
             // end xliff tag
             writer.writeEndDocument();
             writer.flush();
+            writer.close();
```

Comment:
@huangp
No, try with resources won't close its resources first, as long as the try-finally is inside the try with resources.

    try (FileOutputStream fileStream = new FileOutputStream(file)) {
        serializer.setOutputStream(fileStream);
        writer = serializer.getXMLStreamWriter();
        try {
            // use the writer
        } finally {
            closeQuietly(writer);
        }